getting-started/templates/GCP/gcp-supplemental-values.yaml

```diff
@@ -0,0 +1,376 @@
+          - "--structured-logs"
+
+          # Ensures the proxy exits gracefully with a 0 exit code when it receives a SIGTERM signal
+          - "--exit-zero-on-sigterm"
+
+          - "--port=5432"
+          - "<connection-name>" #<ATTENTION> Enter the connection name from the CloudSQL instance
+
+          # The credentials file is required for the proxy to authenticate using a service account key file.
+          - "--credentials-file=/secrets/<secret-key>" #<ATTENTION> Enter the key which was used while creating the secret
+        securityContext:
+          # The default Cloud SQL Auth Proxy image runs as the
+          # "nonroot" user and group (uid: 65532) by default.
+          runAsNonRoot: true
+        restartPolicy: Always
+        # You should use resource requests/limits as a best practice to prevent
+        # pods from consuming too many resources and affecting the execution of
+        # other pods. You should adjust the following values based on what your
+        # application needs. For details, see
+        # https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+        resources:
+          requests:
+            # The proxy's memory use scales linearly with the number of active
+            # connections. Fewer open connections will use less memory. Adjust
+            # this value based on your application's requirements.
+            memory: "1Gi"
+            # The proxy's CPU use scales linearly with the amount of IO between
+            # the database and the application. Adjust this value based on your
+            # application's requirements.
+            cpu: "1"
+
+    ## Extra volumes that can be used in extrContainers
+    extraContainerVolumes:
+      # This volume is required for the proxy to authenticate with cloudSQL using a service account key file. 
+      - name: <token-volume-name> #<ATTENTION> Enter the volume name where the token is to be mounted
+        projected:
+          sources:
+            - serviceAccountToken:
+                audience: <audience-name> #<ATTENTION> Enter the audience name for the projected service account token
+                expirationSeconds: 3600
+                path: token
+      # This volume is required for the proxy to authenticate with cloudSQL when using Workload Identity Federation.
+      - name: <config-volume-name> #<ATTENTION> Enter the volume name where config.json is to be mounted
+        secret:
+          secretName: <secret-name> #<ATTENTION> Enter the secret name where config.json going to be added.
+
+    connectionInfo:
+      ## @param database.connectionInfo.host PostgreSQL hostname. Since the cloud-sql-auth-proxy is used, this value should be localhost.
+      ##
+      host: "localhost"
+      ## @param database.connectionInfo.port PostgreSQL port
+      ##
+      port: "5432"
+      ## @param database.connectionInfo.dbName PostgreSQL database name
+      ##
+      dbName: <database-name>
```

Comment:
Add `<#ATTENTION>` comment.

All values that the user needs to update should have a `<#ATTENTION>` comment. Update it in all the places.